en/docs/get-started/about-this-release.md

```diff
@@ -14,6 +14,10 @@ It is now available to download from [here](https://wso2.com/api-management/#).
 
 ### New features
 
+- **[Exposing an Integration SOAP Service as a Managed API]({{base_path}}/tutorials/integration-tutorials/service-catalog-tutorial-for-proxy-services.md)**
+
+  WSO2 API Manager includes a Service Catalog where developers can register their backend services as Managed APIs. Through the Service Catalog, now SOAP integration services are made discoverable to the API Management layer so that SOAP Passthrough API proxies can directly be created using them.
```

Comment:
```suggestion
  WSO2 API Manager includes a Service Catalog where developers can register their backend services as Managed APIs. Through the Service Catalog, now SOAP integration services are made discoverable to the API Management layer so that SOAP PassThrough API proxies can directly be created using them.
```